test/multiverse/suites/rails/before_suite.rb

```diff
@@ -0,0 +1,23 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+# These are hacks to make the 'rails' multiverse test suite compatible with
+# Rails v7.1 released on 2023-10-05.
+#
+# TODO: refactor these out with non-hack replacements as time permits
```

Comment:
Can you create an issue for this TODO?

Reply:
Done, https://github.com/newrelic/newrelic-ruby-agent/issues/2254